lib/transformations/outputPath/__testfixtures__/outputPath-1.input.js

```diff
@@ -0,0 +1,6 @@
+const path = require('path');
+module.exports = {
+	output: {
+		path: path.join(__dirname, 'dist')
```

Comment:
Oh yeah, what is path is required in the file but `output` doesnnt use it? Need to add some more checks.